example/flutter/objectbox_demo/lib/main.dart

```diff
@@ -43,143 +49,169 @@ class MyHomePage extends StatefulWidget {
   _MyHomePageState createState() => _MyHomePageState();
 }
 
-class _MyHomePageState extends State<MyHomePage> {
-  final _noteInputController = TextEditingController();
+class ViewModel {
   Store _store;
   Box<Note> _box;
-  List<Note> _notes = [];
+  Query<Note> _query;
+
+  ViewModel(Directory dir) {
+    _store = Store(getObjectBoxModel(), directory: dir.path + '/objectbox');
+    _box = Box<Note>(_store);
+
+    final dateProp = Note_.date;
+    final dummyCondition = dateProp.greaterThan(0);
+
+    _query = _box
+        .query(dummyCondition)
```

Comment:
It's not possible to query without conditions?! We should create an issue for that.

Reply:
Agreed.

Reply:
https://github.com/objectbox/objectbox-dart/issues/125